invenio_app_ils/ui/backoffice/README.md

```diff
@@ -0,0 +1,2510 @@
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
I am not sure this belongs here. It could be a link on the readme pointing to react instructions.

Reply:
same :)